chain/chain/src/metrics.rs

```diff
@@ -63,6 +63,13 @@ pub static NUM_ORPHANS: Lazy<IntGauge> =
 pub static HEADER_HEAD_HEIGHT: Lazy<IntGauge> = Lazy::new(|| {
     try_create_int_gauge("near_header_head_height", "Height of the header head").unwrap()
 });
+pub static STARTED_AT_TIMESTAMP: Lazy<IntGauge> = Lazy::new(|| {
+    try_create_int_gauge(
+        "near_started_at_timestamp",
+        "Unix timestamp in seconds of the moment the client was started",
+    )
+    .unwrap()
+});
```

Comment:
Prometheus already defines `node_boot_time_seconds` metric.

Reply:
I'd like to export uptime via telemetry, which needs uptime to be measured by the `neard` process itself.
Context: https://nearinc.atlassian.net/browse/SHRD-130

Happy to rename the new metric to `near_boot_time_seconds` for consistency.